tpm2tools/flush_active_handles.go

```diff
@@ -0,0 +1,40 @@
+package tpm2tools
+
+
+import (
+    "io"
+    "github.com/google/go-tpm/tpm2"
+    "github.com/google/go-tpm/tpmutil"
+)
+
+// FlushActiveHandles calls FlushContext() on all active handles within the 
```

Comment:
Here and elsewhere the correct terminology is "transient handles" instead of "active handles", as there are actually multiple types of handles we might want to clear:
  - Transient Handles (`TPM_HT_TRANSIENT`)
  - Loaded Session Handles (`TPM_HT_LOADED_SESSION`)
  - Saved Session Handles (`TPM_HT_SAVED_SESSION`)

Reply:
Implemented in standalone binary.